cluster/engine.go

```diff
@@ -184,6 +185,14 @@ func (e *Engine) StartMonitorEvents() {
 func (e *Engine) ConnectWithClient(client dockerclient.Client) error {
 	e.client = client
 
+	// Use HTTP Client used by dockerclient to create engine-api client
```

Comment:
Yep. I couldn't get `mock` client working, even though it looks easy. I'll change this if I can.